css/properties/text-underline-offset.json

```
@@ -36,7 +36,7 @@
               "version_added": false
             },
             "opera": {
-              "version_added": false
+              "version_added": true
```

```suggestion
              "version_added": "73"
```

Setting this to 73 will satisfy the linter. I might have assumed this based on the Chrome to Opera version mapping, but the BrowserStack link was very helpful to confirm!